Sources/XCTest/XCTestCase.swift

```diff
@@ -45,21 +55,19 @@ extension XCTestCase {
                 tearDown()
 
                 totalDuration += duration
-                for failure in XCTCurrentFailures {
+                for failure in failures {
                     failure.emit(method)
                     totalFailures += 1
                     if !failure.expected {
                         unexpectedFailures += 1
                     }
+
+                    let result = failures.count > 0 ? "failed" : "passed"
+
+                    print("Test Case '\(method)' \(result) (\(printableStringForTimeInterval(duration)) seconds).")
```

Comment:
I believe this is a regression that causes "passed" to never be output.

Before, the following line appeared **outside** of the `XCTCurrentFailures` loop:

```swift
print("Test Case '\(method)' \(result) (\(printableStringForTimeInterval(duration)) seconds).")
```

Now, it appears **within** the failures for loop. As a result, only the results of *failing* tests are printed.